lib/Map/RegionProvider.js

```diff
@@ -116,6 +117,17 @@ var RegionProvider = function(regionType, properties) {
      * @type {Object}
      */
     this._idIndex = {};
+
+    /**
+     * The URL of a pre-generated JSON file containing just a long list of IDs for a given
+     * layer attribute, in the order of ascending feature IDs (fids). If defined, it will
+     * be used in preference to requesting those attributes from the WFS server.
+     * @type {String}
+     */
+    this.regionIdList = properties.regionIdList;
```

Comment:
Actually on closer examination, am I right that this URL returns a json object eg. {"layer": "region_map:FID_SA1_2011_AUST", "property": "SA1_MAIN11", "values": [10101, ...]} ?  I'd have guessed something named `regionIdsListUrl` would return an array, not an object - or perhaps an object with a property called `regionIdsList`. 
So, what would you say to renaming `values` to `ids` in the json, and renaming this to `this.regionIdsUrl`?
Of course, I realise I'm probably being too pedantic. :-)